content/en/lotus/developers/contribute.md

```diff
@@ -36,9 +36,9 @@ But the _Help Wanted_ label also exists in several related projects:
 
 The Filecoin community is active and here to answer your questions in your channel of choice.
 
-For shorter-lived discussions, our community chat is open to all on [Slack](https://filecoin.io/slack/) in the #fil-lotus-dev channel.
+For shorter-lived discussions, our community chat is open to all on [Slack](https://filecoinproject.slack.com/) in the #fil-lotus-dev channel.
```

Comment:
The updated address (https://filecoinproject.slack.com) only allows users that already have Filecoin Slack accounts to log-in, the correct link is the link https://filecoin.io/slack, which redirects users with a invite link, so that they can create Filecoin Slack accounts. I therefore suggest reverting:

```suggestion
For shorter-lived discussions, our community chat is open to all on [Slack](https://filecoin.io/slack/) in the #fil-lotus-dev channel.
```